microsoftgraph/microsoft-graph-explorer-v4
# Feature: Add Adaptive cards template JSON code

## Overview

Add a section on the Adaptive Cards Tab that displays the JSON code used to render the card.

Fixes #199 

### Demo

![image](https://user-images.githubusercontent.com/18407044/106869390-ed027200-66e0-11eb-8298-0e4ef39284c4.png)

![image](https://user-images.githubusercontent.com/18407044/107006412-251db980-67a2-11eb-82d3-37a41c8d5c62.png)


### Notes

Opted to use tabs so as to not distort the size and display of the cards. As opposed to displaying the cards and the JSON code side by side.

## Testing Instructions
* Run a sample query
* Navigate to the Adaptive Cards Tab
* If a card is present, observe a 'Card' tab and 'Template JSON' tab
* Navigate to the 'JSON Schema' tab and observe the template used to render the code.
